index.html

```diff
@@ -1256,8 +1256,96 @@
       <section>
         <h3>Usage Patterns</h3>
 
-        <p class="issue">
-When verifiable claims are used can be used for correlation purposes.
+        <p>
+Despite the best efforts to assure privacy, the actual use of verifiable
+claims can potentially lead to de-anonymization and a loss of privacy.
+This correlation can occur:
+        </p>
+
+        <ol>
+          <li>
+When the same claim is presented to the same inspector more than once –
+that inspector could infer that the holder is the same individual.
+          </li>
+          <li>
+When the same claim is presented to different inspectors, and either those
+inspectors collude or a third party has access to transaction records from
+both inspectors – the observant party could infer that the individual
+presenting the claims is the same person at both services, i.e., the
+accounts are controlled by the same person.
+          </li>
+          <li>
+When the same subject ID of a claim refers to the same subject across
+presentations or inspectors. Even when different claims are presented,
+if the subject ID is the same, inspectors (and those with access to
+inspector logs) could infer that the holder of the claims is the same
+person.
+          </li>
+          <li>
+When the underlying information in a claim can be used to identify an
+individual across services – using information from other sources
+(including information provided directly by the user), inspectors can use
+the information inside the claim to correlate the individual with an
+existing profile. For example, if a holder presents claims that include
+zip code, age, and income, the inspector can potentially correlate the
+subject of that claim with an established profile which already contains
+the subject’s zip code, age, and income.
+          </li>
+          <li>
+When passing the ID of a claim to a centralized revocation server – the
+centralized server can correlate the claim usage across interactions. For
+example, if a verifiable claim is used for proof of age in this manner, the
+centralized service could know everywhere that claim was presented: all
+liquor stores, bars, adult stores, lottery purchases, etc.
+          </li>
+        </ol>
+
+        <p>
+It’s possible to mitigate this in part:
+        </p>
+
+        <ol>
+          <li>
+Use a globally unique id as the subject for any given claim and never
```

Comment:
Not necessarily. It depends on the rest of the credential (or grouped claims). You could have multiple attributes of the same subject correlated within a claim group, with a unique Subject ID that is never used elsewhere, e.g., 
```
AF48 a person;
         named Joe;
         hasDog Fido;
         hasHeight 5.75;
         hasFriend Bob3712.
```
If AF48 is understood to have no meaning elsewhere, then the correlations *within* the group of claims are all that are exposed by the claim group. Of course, the claims themselves may allow de-anonymization, but that's a different issue.

Perhaps this is semantic confusion around a single claim, i.e., a single signed attribute, verses a group of claims. I can see that if the ID is unique on a per-attribute basis, it's essentially useless. When I wrote it, I was being sloppy about treating bundled claims as claims.

But then, if the data model makes subject IDs optional, then it clearly allows multiple claims to be communicated with an implied subject, which is closer to what I intended to suggest.

Either way, we need to be more rigorous.